numba/cuda/tests/cudapy/test_exception.py

```diff
@@ -91,6 +93,56 @@ def test_raise_causing_warp_diverge(self):
         """
         self.case_raise_causing_warp_diverge(with_debug_mode=False)
 
+    # The following two cases relate to Issue #7806: Division by zero stops the
+    # kernel. https://github.com/numba/numba/issues/7806.
+
+    def test_no_zero_division_error(self):
+        # When debug is False:
+        # - Zero by division raises no exception
```

Comment:
```suggestion
        # - Division by zero raises no exception
```

Reply:
Thanks, not quite sure how I managed to write that!